app/Http/Controllers/PraktikanController.php

```diff
@@ -110,6 +111,27 @@ public function update(Request $request, Praktikan $praktikan)
         //
     }
 
+    public function resetPass(Request $request){
+        $user = Auth::guard('praktikan')->user();
+        $request->validate([
+            'password' => 'required|min:6',
+            'repeatpass' => 'required|min:6',
+        ]);
+        if(strcmp($request->password, $request->repeatpass)==0){
```

Comment:
you dont have to put `== 0` check here as when it return 0, then you will know that its correct so it should be
```suggestion
        if(!strcmp($request->password, $request->repeatpass)){
```